ci/upload_to_owncloud.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+###############################################################
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# Copyright (c) 2020 Tomer Eliyahu (Intel)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+###############################################################
+
+scriptdir="$(cd "${0%/*}" && pwd)"
+rootdir=$(realpath "$scriptdir/../")
+
+# shellcheck source=functions.sh
+. "$rootdir/tools/functions.sh"
+
+usage() {
+    echo "usage: $(basename "$0") [-hv] <user> [user] <password> [password] <remote-path> [path] <local-path> [path]"
```

Comment:
I read this as: you can call it as `upload_to_owncloud.sh arnout arnoutpassword path/on/server ./file` or as `upload_to_owncloud.sh arnout user arnoutpassword password path/on/server path ./file path` - which makes no sense.

In other words:
```suggestion
    echo "usage: $(basename "$0") [-hv] <user> <password> <remote-path> <local-path>"
```